pkg/apis/machine/v1beta1/machine_webhook.go

```diff
@@ -84,8 +87,15 @@ var (
 )
 
 const (
-	defaultUserDataSecret  = "worker-user-data"
-	defaultSecretNamespace = "openshift-machine-api"
+	DefaultWebhookName                  = "machine-api-operator-webhook"
```

Comment:
defaultUserDataSecret and defaultNamespace are values we choose to default but might vary. The rest of this variable values are unique. We don't need to prepend them with `Default`.

Also do they need to be exported?

Reply:
https://github.com/openshift/machine-api-operator/pull/626/files/0174c45fa8abf5eab17e7b487f66aa01df66fe55#diff-98bc71b7347ada0763f9e57ae6c9d235R90-R91 should not be. The rest of them are used in `sync.go`